src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -6484,5 +6841,63 @@ public static async Task M1()
 
             VerifyOperationTreeAndDiagnosticsForTest<BlockSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void UsingDeclaration_DefaultDisposeArguments()
+        {
+            string source = @"
+class C
+{
+    public static void M1()
+    {
+        /*<bind>*/using var s = new S();/*</bind>*/
```

Comment:
>using var s = new S(); [](start = 18, length = 22)

Do we have a CFG coverage for using declarations vs. using statements?

Reply:
Added `UsingDeclaration_Flow_22`

---
In reply to: [539537373](https://github.com/dotnet/roslyn/pull/49842#discussion_r539537373) [](ancestors = 539537373)